apps/studio/data/edge-functions/edge-functions-deploy-mutation.ts

```diff
@@ -65,7 +65,7 @@ export const useEdgeFunctionDeployMutation = ({
         const { projectRef, slug } = variables
         await Promise.all([
           queryClient.invalidateQueries(edgeFunctionsKeys.list(projectRef)),
-          queryClient.invalidateQueries(edgeFunctionsKeys.body(projectRef, slug)),
+          queryClient.invalidateQueries(edgeFunctionsKeys.detail(projectRef, slug)),
```

Comment:
Invalidating `detail` query here will ensure `Details` tab is updated with new deploy. Also, looks like invalidating `detail` query will also trigger a new request to body (I'm not sure exactly how that happens though) cc: @SaxonF

Reply:
^ at first glance - we probably need to keep the body invalidation so that the code content is updated, will look into why invalidating details also triggers the body to invalidate

Reply:
we probably don't need the list invalidation actually